api/src/main/kotlin/edu/wgu/osmt/db/CollectionStatusEnum.kt

```diff
@@ -0,0 +1,16 @@
+package edu.wgu.osmt.db
```

Comment:
Since Kotlin allows multiple classes / interfaces / enums per file, I think it makes sense OK to combine this source file with PublishStatus.kt, and organize the constants together.